www/data_service/src/classes/build.service.coffee

```diff
@@ -0,0 +1,15 @@
+class Build extends Factory
+    constructor: (Base, dataService) ->
+        return class BuildInstance extends Base
+            constructor: (o, e) ->
+                super(o, e)
+
+                endpoints = [
+                    'changes'           # /changes
+                    'properties'        # /properties
+                    'steps'             # /steps/:name
+                                        # /steps/:stepid
```

Comment:
I like the idea. I think we can go further, and create the endpoint with more details , in order to allow argument checks

           endpoints = [
                    'changes' 
                    'properties' 
                   'steps/:name_orid'
           ]